libs/framework/middlewares/src/prerender.tsx

```diff
@@ -85,13 +85,17 @@ export const prerenderMiddleware = remoteEntry => {
         </ReduxProvider>
       )
 
+      const initialState = res?.initialState ? JSON.stringify(res.initialState) : 'NO STATE'
+
       const {pipe} = renderToPipeableStream(el, {
         onAllReady() {
           // If something errored before we started streaming, we set the error code appropriately.
           res.statusCode = didError ? 206 : 200
           res.contentType('text/plain')
           res.write(stringifiedChunks)
           res.write(constants.SERIALISED_RESPONSE_SEPARATOR)
+          res.write(initialState)
```

Comment:
I think we probably want this to be after `pipe(res)` so it's not a breaking change for consumers who are splitting the response to an array and destructuring it, e.g. 
```
      const [chunks, html] = res.data.split(constants.SERIALISED_RESPONSE_SEPARATOR)
```